### DISCUSSION TITLE
is there a way to reference another post with preview and short description?

### DISCUSSION THREAD

**Discussion by sarkrui** [adityatelange/hugo-PaperMod#524] [Q&A]
Quick question, is there a way to reference another post with preview and short description, like Notion and Quora. 
- [Demo 1 ](https://www.notion.so/fine-biplane-80a/Can-a-jailbreak-tweak-be-created-to-run-Mac-apps-on-an-M1-iPad-Pro-considering-it-has-all-the-necess-a48d7bfd84f649b79f5742cb7156bed0)
- [Demo 2](https://www.quora.com/Can-a-jailbreak-tweak-be-created-to-run-Mac-apps-on-an-M1-iPad-Pro-considering-it-has-all-the-necessary-hardware-components-to-do-so)
![image](https://user-images.githubusercontent.com/14228455/129921050-e6351ee5-3bad-4485-843b-3bd2e6d13ce0.png)

Or if anyone has created a shortcode for similar purpose? Please do share, thank you!

**Comment by RoneoOrg** [comment]
Bookmarked but not tested so far: https://blog.jeremylikness.com/blog/create-article-preview-in-hugo/
